pysqueezebox/player.py

```diff
@@ -785,14 +785,21 @@ async def async_load_playlist(
 
         if cmd == "insert":
             for item in reversed(playlist):
+                if not item.get("url"):
+                    continue
```

Comment:
```suggestion
```
The url key is required, right? I'm curious why you added this check here.

If it's needed we could instead validate the playlist once at the beginning of the function rather than repeating it below for each type of cmd.

Reply:
There are some really odd responses from some apps for non-playable items (text documents, album synopsis - all sorts of stuff shows up).  So, these checks are just to stop it failing when it gets an item it can't process.  We can't identify them individually as each app does different things.  This is basically just to throw away weird stuff.

Reply:
Ah, got it. Thanks.

Reply:
There doesn't seem to be any standard about what content the apps return.  I had to get Michael to add a fix to the qobuz app to make it work at all with the cli and even so, it works differently to the tidal app, which is different to spotify...